CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - **Breaking:** On Web, remove the `stdweb` backend.
 - Added `Window::focus_window`to bring the window to the front and set input focus.
+- Prevent a panic on interrupted system calls on the `x11` backend.
```

Comment:
```suggestion
- On X11, prevent a panic on interrupted system calls.
```
Could you move this to the top of the changelog?

Reply:
Done 😉